warn/docs/warnings.textproto

```diff
@@ -480,15 +480,128 @@ warnings: {
 }
 
 warnings: {
-  name: "native-cc"
+  name: "native-cc-binary"
   header: "All C++ build rules should be loaded from Starlark"
-  description:
-    "The CC build rules should be loaded from Starlark.\n\n"
-    "Update: the plans for disabling native rules\n"
-    "[have been postponed](https://groups.google.com/g/bazel-discuss/c/XNvpWcge4AE/m/aJ-aQzszAwAJ),\n"
-    "at the moment it's not required to load Starlark rules."
-  bazel_flag: "--incompatible_load_cc_rules_from_bzl"
-  bazel_flag_link: "https://github.com/bazelbuild/bazel/issues/8743"
+  description: "The CC build rules should be loaded from @rules_cc."
+  autofix: true
+}
+
+warnings: {
+  name: "native-cc-common"
```

Comment:
These warnings are equivalent, please compress them into a single item (just use multiple `name: ...` lines for the same warning, see `"function-docstring"` in the same file below for an example).

Reply:
Done, thanks!